### ISSUE TITLE
Test leads to image diseapearing

### ISSUE THREAD

**Issue by loiclavoie** [voidlabs/mosaico#240]
When I send a test email, every image display within the right pane diseapear. 

Their SRC is still valid, and when I drag and drop them up and down, the images reappear. They also appear correctly within the email received.

Is there a refresh method I should call or something like that to "re-render" the left pane post test?

**Comment by bago** [comment]
what browser?
what mosaico version? 
what backend?
did you see the same behaviour on "http://mosaico.io" demo?

by "right pane" do you mean the "mobile preview" pane or the main working area?

**Comment by loiclavoie** [comment]
Chrome (Dev - latest version)
Mosaico - branch master
PHP custom (based on the one provided in your doc)
No, the problem is not on the live site. I have the feeling it might be linked to the generation of URL since that's the only thing I changed.

By right pane, I mean the edition panel.

The only thing I changed is the URL ko.bindingHandlers.wysiwygSrc.convertedUrl

Somehow, the system was sending email with the resize path instead of the return path. The url of the images was looking like this
www.pathtobackend.org/src="path to image"&method=resize&width=...

Basically, it was relaunching the method of resize (skipped over in the backend when the image was already existing), but it was slow like hell. The system should just take the src received as is. 
So, I feel like when the test is sent to the backend again, it tries to recall the old resize adress instead of the new one. Hence I've replace the function.

**Comment by bago** [comment]
Hi, I'm not sure I understand ... 

If you don't change the code then you don't have issues: right?

You only reproduce this issue when you replace the "convertedUrl" method: is it correct?

Then, can you share your method and try to better explain what's going on? Do you think Is there a bug in Mosaico or in your "convertedUrl" method? or in your backend?

**Comment by bago** [comment]
Hi, I'm not sure I understand ... 

If you don't change the code then you don't have issues: right?

You only reproduce this issue when you replace the "convertedUrl" method: is it correct?

Then, can you share your method and try to better explain what's going on? Do you think Is there a bug in Mosaico or in your "convertedUrl" method? or in your backend?

**Comment by bago** [comment]
So, what was the issue? Please report back so this won't be a waste of time for us given than others will be able to find what fixed your issue.

**Comment by loiclavoie** [comment]
I haven't figured out what was the issue. I don't really have the time to investigate further at this point.

**Comment by benj0c** [comment]
I have the same issue. When click on send preview, some of image requests are cancelled. This is screenshot  from "http://mosaico.io" demo.
![Screen Shot 2019-11-26 at 1 41 50 PM](https://user-images.githubusercontent.com/4091095/69645497-8dcdba00-1066-11ea-961a-6e98de33a500.png)

Do you know why this happens?

**Comment by benj0c** [comment]
@bago any information?

**Comment by benj0c** [comment]
This happens only in Chrome and Brave browsers (same engine - chromium). For some reason these browsers cancel some requests and that leads image disappearing.

@bago any idea?
